### ISSUE TITLE
selection bug in text.go

### ISSUE THREAD

**Issue by rjkroege** [rjkroege/edwood#51]
On double-clicking and not a Frame issue. 😊 

```
Text.Select: DoubleClick returned 27783, 0
panic: acme: textsetselect p0=518 p1=0 q0=27783 q1=0 t.org=27265 nchars=1134
goroutine 11 [running, locked to thread]:
main.(*Text).SetSelect(0xc420458528, 0x6c87, 0x0)
        /Users/rjkroege/tools/gopkg/src/github.com/rjkroege/edwood/text.go:1303 +0x352
main.(*Text).Select(0xc420458528)
        /Users/rjkroege/tools/gopkg/src/github.com/rjkroege/edwood/text.go:1118 +0x670
main.MovedMouse(0x4c6, 0x241, 0x1, 0x2b2ef9bc)
        /Users/rjkroege/tools/gopkg/src/github.com/rjkroege/edwood/acme.go:418 +0x4fb
main.mousethread(0xc4200b60c0)
        /Users/rjkroege/tools/gopkg/src/github.com/rjkroege/edwood/acme.go:297 +0x31c
created by main.main
        /Users/rjkroege/tools/gopkg/src/github.com/rjkroege/edwood/acme.go:171 +0x934
```

**Comment by rjkroege** [comment]
Very difficult to replicate btw: this happened once over the course of several days of use.

**Comment by rjkroege** [comment]
Likely fixed in 9decce2a49f0004a2a56401e1a2c2dbf4a65a186
